scripts/spell_check/check_spelling.sh

```diff
@@ -41,6 +41,9 @@ shift $(expr $OPTIND - 1)
 if [ ! $# -eq 0 ]; then
   EXCLUDE=$(cat $AGIGNORE | sed -e 's/\s*#.*$//' -e '/^\s*$/d' | tr '\n' '|' | sed -e 's/|$//')
   INPUTFILES=$(echo $@ | tr -s '[[:blank:]]' '\n' | egrep -iv "$EXCLUDE" | tr '\n' ' ' )
+  if [ "$INPUTFILES" == "" ]; then
+    exit 0
+  fi
```

Comment:
Yes, it you don't specify anything it's the test at line 41 who will going to select all the files, this test is useful when we specify only filtered files :-)